Makefile

```diff
@@ -115,7 +115,8 @@ test-e2e: ##@tests Run e2e tests
 	go test -timeout 20m ./e2e/rpc/... -network=$(networkid)
 	go test -timeout 20m ./e2e/whisper/... -network=$(networkid)
 	go test -timeout 10m ./e2e/transactions/... -network=$(networkid)
-	go test -timeout 40m ./lib -network=$(networkid)
+	# e2e_test tag is required to include some files from ./lib without _test suffix
```

Comment:
Maybe put issue URL here so anyone looking at this comment will know where to look more explanation?

Reply:
I added a comment in `lib/utils.go` which may explain it even better than the issue :D